src/plugins/condition/components/Condition.vue

```diff
@@ -50,11 +50,14 @@
             <span class="c-condition__name">{{ condition.configuration.name }}</span>
             <span class="c-condition__summary">
                 <template v-if="!canEvaluateCriteria">
-                    Define criteria
+                    <span v-if="!condition.isDefault">
```

Comment:
This should use <template> rather than a span to avoid unneeded markup.

Reply:
I don't think a template tag can have a template tag as its immediate parent. I once tried this at the top level and it didn't work. See here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template
> 

Permitted parents | Any element that accepts metadata content, phrasing content, or script-supporting elements. Also allowed as a child of a colgroup element that does not have a span attribute.
-- | --

Reply:
So I just tried it in the code and it works fine. I just pushed the change back up.

Reply:
In the branch @shefalijoshi merged this into she found a more elegant way to do it without the direct template/template nesting.

Reply:
```
<span class="c-condition__summary">
    <template v-if="!condition.isDefault && !canEvaluateCriteria">
        Define criteria
    </template>
<span v-else>
    <condition-description :show-label="false"
                           :condition="condition"
    />
</span>
```